build.gradle.kts

```diff
@@ -13,6 +15,9 @@ kotlin {
             dependencies {
                 implementation("org.apache.kafka:connect-api:2.6.0")
                 implementation("org.apache.kafka:kafka-clients:2.6.0")
```

Comment:
I'm curious if this works with `compileOnly(...)` for these dependencies (not bundling them, as they're already provided).

Reply:
Oh! Uh. Is this the right version of the libraries? We have `2.4.1`, as far as I know.